src/components/Notiflication/Notiflication.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { TodoStatus } from '../../types/TodoStatus';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+interface FooterProps {
+  todos: Todo[];
+  selectedStatus: TodoStatus;
+  onStatusChange: (status: TodoStatus) => void;
+  filteringTodosByActiveStatus: number;
+  TodoStatusRoutes: Record<TodoStatus, string>;
+}
+
+export const Footer: React.FC<FooterProps> = ({
+  todos,
+  selectedStatus,
+  onStatusChange,
+  filteringTodosByActiveStatus,
+  TodoStatusRoutes,
+}) => {
+  if (todos.length > 0) {
+    return (
```

Comment:
You duplicated footer instead of error notification component
<img width="595" alt="image" src="https://github.com/user-attachments/assets/2f8ef10d-c83d-4f85-a3d6-3df82e86f574">